pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -4690,7 +4883,7 @@ spec:
 	}
 }
 
-func TestReconcileWithPipelineResults(t *testing.T) {
+func TestReconcileWithFinallyResults(t *testing.T) {
```

Comment:
it seems that this change is unrelated 🤔

Reply:
Ah, right - so `TestReconcileWithPipelineResults` was calling both of `runTestReconcileWithFinallyResults` _and_ `runTestReconcileWithPipelineResults`. I had to fix some testcases/code in those, and it was really confusing as to what exactly was failing, so while I was here, I split them up to actually make sense. Unrelated, yeah, but might as well do it here.

Reply:
I see, thanks for the clarification!